tests/ZeroC.Slice.Tests/CustomTypeTests.cs

```diff
@@ -9,32 +10,207 @@ public class CustomTypeTests
     [Test]
     public void Decode_custom_type()
     {
+        // Arrange
```

Comment:
Since we support custom types with both Slice1 and Slice2, this test should test Slice1 and Slice2, not just Slice2.
```
[Test]
public void Decode_custom_type([Values]SliceEncoding sliceEncoding)
...
```